packages/sdk/src/client/client.ts

```diff
@@ -164,6 +171,7 @@ const DefaultClientOptions = {
   syncLoopDuration: 50,
   retrySyncLoopDelay: 1000,
   reconnectStreamDelay: 1000,
+  reconnectionStreamTimeout: 60000,
```

Comment:
Comment says that the default value for `reconnectionStreamTimeout` is `30000`, but it is actually set to `60000`. Is `30000` the intended value?

```suggestion
  reconnectionStreamTimeout: 30000,
```

Reply:
Oh, I'm currently testing to determine the best timeout value for this mechanism.

Reply:
I'm currently testing to determine the optimal timeout value for this mechanism. The values will be finalized before this PR is opened for review.